gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -114,32 +157,84 @@ def call_quit(self, _):
         """Process Quit command - quit the program"""
         return True
 
-    def call_cg(self, _):
-        if pycoingecko_controller.menu():
-            return True
-        print("")
+    def call_load(self, other_args):
+        """Process load command"""
+        try:
+            self.current_coin, self.source = load(
+                coin=self.current_coin, other_args=other_args
+            )
+        except TypeError:
+            print("Couldn't load data\n")
+
+    def call_chart(self, other_args):
+        """Process chart command"""
+        if self.current_coin:
+            getattr(self.DD_VIEWS_MAPPING[self.source], "chart")(
+                self.current_coin, other_args
+            )
+        else:
+            print("No coin selected. Use 'load' to load the coin you want to look at.")
+            print("")
```

Comment:
very small formatting point, but whenever you have a `print("")`, just try to add a `"\n"` to a previous print if there is one